.github/workflows/ci.yml

```diff
@@ -6,22 +6,11 @@ on:
   pull_request:
 
 jobs:
-  lint:
-    strategy:
-      fail-fast: false
-
-    name: Lint
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v4
-      - run: make lint
-
```

Comment:
Note to reviewers: I intentionally removed the linting workflow here, since it was depending on the ROS 2 devcontainer to work.  While not having linting on this repository is unfortunate, given that this is in maintenance mode I don't think we'll have a huge amount of change, and thus I think it is acceptable to go without linting here.